README.md

```diff
@@ -86,6 +97,7 @@ There are three particular `rspec-rails` specific changes to be aware of:
 3. [Rails 4.x `ActiveRecord::Migration` pending migration checks](https://www.relishapp.com/rspec/rspec-rails/docs/upgrade#pending-migration-checks)
 4. Extraction of `stub_model` and `mock_model` to
    [`rspec-activemodel-mocks`](https://github.com/rspec/rspec-activemodel-mocks)
+5. In Rails 5.x, controller testing has been moved to its own gem which is [rails-controller-testing](https://github.com/rails/rails-controller-testing). So using `assings` in your controller test without adding this gem would raise an exception. 
```

Comment:
You have a typo here, `assings` instead of `assigns` and would you mind re-wording it slightly? It reads better as 
> Using `assigns` in your controller specs without adding this gem will no longer work.